indico/modules/events/management/controllers/protection.py

```diff
@@ -63,42 +70,86 @@ def _process(self):
                                 endpoint='event_management.acl')
 
 
-class RHEventProtection(RHManageEventBase):
+class RHEventProtectionBase(RHManageEventBase):
+    def __init__(self):
+        super(RHEventProtectionBase, self).__init__()
+        selectable_permissions = {k: v for k, v in get_available_permissions(Event).viewitems() if v.user_selectable}
+        self.whitelisted_permissions = set(selectable_permissions.keys()) | {'edit', 'access'}
```

Comment:
calling `.keys()` is pretty much never necessary. Iterating a dict gives you the keys already, so just `set(selectable_permissions)` is enough.